airflow/www/views.py

```diff
@@ -578,7 +578,8 @@ def _get_logs_with_metadata(try_number, metadata):
         try:
             if ti is not None:
                 dag = dagbag.get_dag(dag_id)
-                ti.task = dag.get_task(ti.task_id)
+                if dag:
```

Comment:
```suggestion
                if ti and dag:
```

Reply:
so you can remove `if ti is not None:`

Reply:
I am not sure. There may be a task instance without DAG or DAG without task. These are the correct cases. Please note that dag variable is None, when task is not found.

Reply:
I got it. 👍